specification/orbital/resource-manager/Microsoft.Orbital/stable/2022-11-01/orbital.json

```diff
@@ -0,0 +1,2264 @@
+          "description": "Longitude of the ground station in decimal degrees."
+        },
+        "latitudeDegrees": {
+          "type": "number",
+          "description": "Latitude of the ground station in decimal degrees."
+        },
+        "altitudeMeters": {
+          "type": "number",
+          "description": "Altitude of the ground station."
+        },
+        "releaseMode": {
+          "type": "string",
+          "enum": [
+            "Preview",
+            "GA"
+          ],
+          "x-ms-enum": {
+            "name": "releaseMode",
+            "modelAsString": true
+          },
+          "description": "Release Status of a ground station."
+        }
+      },
+      "description": "Properties object for Available groundstation."
+    },
+    "Capability": {
+      "type": "string",
+      "description": "Capability of the Ground Station.",
+      "enum": [
+        "EarthObservation",
+        "Communication"
+      ],
+      "x-ms-enum": {
+        "name": "Capability",
+        "modelAsString": true
+      }
+    },
+    "ResourceIdListResult": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "id": {
+                "type": "string",
+                "description": "The Azure Resource ID."
+              }
+            }
+          },
+          "description": "A list of Azure Resource IDs."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URL to get the next set of results.",
+          "readOnly": true
+        }
+      },
+      "description": "Response for an API service call that lists the resource IDs of resources associated with another resource."
+    },
+    "Etag": {
+      "readOnly": true,
+      "type": "string",
+      "description": "A unique read-only string that changes whenever the resource is updated."
+    },
+    "ProvisioningState": {
+      "type": "string",
+      "enum": [
+        "Creating",
+        "Succeeded",
+        "Failed",
+        "Canceled",
+        "Updating",
+        "Deleting"
+      ],
+      "x-ms-enum": {
+        "name": "ProvisioningState",
+        "modelAsString": true
+      },
+      "description": "The current state of the resource's creation, deletion, or modification.",
+      "readOnly": true
+    },
+    "AuthorizedGroundstation": {
+      "type": "object",
+      "required": [
+        "groundStation",
+        "expirationDate"
+      ],
+      "readOnly": true,
+      "properties": {
+        "groundStation": {
+          "type": "string",
+          "description": "Groundstation name."
+        },
+        "expirationDate": {
+          "type": "string",
+          "format": "date",
+          "description": "Date of authorization expiration."
+        }
+      },
+      "description": "Authorized groundstation."
+    },
+    "OperationResult": {
```

Comment:
May I suggest you make this definition inherit OperationResult from common types using 'allOf'?
If its not obviously breaking...

In general,
The OperationResult type should be defined correctly in commontypes types.json. It is recommended to reuse the common definition instead of declaring your own from scratch. [Otherwise it is easy to declare incorrectly.]
```json
            "schema": {
              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/OperationResult"
            }
```